data/projects.json

```
@@ -39,6 +39,16 @@
         "site": "https://roshaen.github.io/currency",
         "tags": "html, css, js"
     },
+    {
+        "name": "eLearning",
+        "display": "eLearning",
+        "description": "It's a simple landing page. Made with Bootstrap 5.",
+        "author": "hasan-naim",
+        "category": "website",
+        "level": "basic",
+        "site": "https://elearning-hasan.netlify.app/",
+        "tags": "html, css, bootstrap"
```

Add `e-learning` tag for groundation of the project
```suggestion
        "tags": "e-learning, html, css, bootstrap"
```